crates/ruff/src/jupyter/schema.rs

```diff
@@ -0,0 +1,256 @@
+//! The JSON schema of a Jupyter Notebook, entrypoint is [`JupyterNotebook`]
+//!
+//! Generated by <https://app.quicktype.io/> from
+//! <https://github.com/jupyter/nbformat/blob/16b53251aabf472ad9406ddb1f78b0421c014eeb/nbformat/v4/nbformat.v4.schema.json>
+//! Jupyter Notebook v4.5 JSON schema.
+//!
+//! Compared to the generated version, `Cell::id` is optional because it wasn't required <v4.5,
+//! I added `#[serde(deny_unknown_fields)]` where the schema had `"additionalProperties": false`
```

Comment:
For whatever reason I tend to use "we" instead of "I" in code comments, I don't know why :joy:

Reply:
I guess to me it conveys that we're all responsible for whatever is being written / committed

Reply:
i normally use we for general explanations and i when i did something opionated, but i agree with the ~~royal~~ [mathematical](https://math.stackexchange.com/a/447722) we over i. here i think we can get away with passive voice.